env0/resource_sshkey.go

```diff
@@ -83,16 +71,19 @@ func resourceSshKeyDelete(ctx context.Context, d *schema.ResourceData, meta inte
 }
 
 func resourceSshKeyImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	return nil, errors.New("Not implemented")
-	// apiClient := meta.(*client.ApiClient)
-
-	// id := d.Id()
-	// ssh key, err := apiClient.SshKey(id)
-	// if err != nil {
-	// 	return nil, err
-	// }
-
-	// d.Set("name", ssh key.Name)
-
-	// return []*schema.ResourceData{d}, nil
+	id := d.Id()
+	var getErr diag.Diagnostics
+	_, uuidErr := uuid.Parse(id)
+	if uuidErr == nil {
+		_, getErr = getSshKeyById(id, meta)
+	} else {
+		var sshKey client.SshKey
+		sshKey, getErr = getSshKeyByName(id, meta)
```

Comment:
We should log wether we query by name or id

Reply:
https://github.com/env0/terraform-provider-env0/pull/40/commits/7a2a7d32dce0fe8f8505200d619926a53823be29